lib/features/composer/presentation/widgets/email_address_input_builder.dart

```diff
@@ -309,12 +320,16 @@ class EmailAddressInputBuilder {
 
   Widget _buildCounter(BuildContext context, int count) {
     return Padding(
-      padding: const EdgeInsets.only(left: 8, top: kIsWeb ? 8 : 0),
+      padding: const EdgeInsetsDirectional.only(start: 8, top: PlatformInfo.isWeb ? 8 : 0),
       child: InkWell(
         onTap: () => _onShowFullListEmailAddressAction?.call(_prefixEmailAddress),
         child: Chip(
           labelPadding: const EdgeInsets.symmetric(horizontal: 8),
-          label: Text('+$count', maxLines: 1, overflow: kIsWeb ? null : TextOverflow.ellipsis),
+          label: Text(
+            '+$count',
```

Comment:
why dont you make using L10n ?

Reply:
Oh, I don't think we need L10n here. It is just 1 lable showing the quantity. :))